### DISCUSSION TITLE
How to provide a different icon for expand and collapse ?

### DISCUSSION THREAD

**Discussion by yaswanthmaddula-sureify** [CarlosNZ/json-edit-react#114] [Q&A] [answered]
I see that the library exposed `chevron` but how to manage its expand and collapse state ?

**Comment by CarlosNZ** [comment]
It's the same icon, just rotated. You can't currently specify a different icon for each state (and I'm not totally sure why you would want to?)

**Comment by yaswanthmaddula-sureify** [comment]
I added a different icon and its not rotating itself on collapse. @CarlosNZ

**Comment by CarlosNZ** [comment]
Ah yes, good point. I've created a bug issue to sort this: https://github.com/CarlosNZ/json-edit-react/issues/115

Thanks for reporting, will fix ASAP

**Comment by CarlosNZ** [comment, marked as answer]
This should be fixed now in [v1.15.11](https://www.npmjs.com/package/json-edit-react/v/1.15.11)
